integration-tests/utils/integration-utils.js

```diff
@@ -29,37 +29,39 @@ const findInGrid = async (page, text) => {
   return await page.waitForXPath(`//*[@role="grid"][contains(normalize-space(.),"${text}")]`)
 }
 
-const exactlyFindClickable = (page, text) => {
-  return page.waitForXPath(`(//a | //*[@role="button"] | //button)[normalize-space(.)="${text}"]`)
+const clickable = ({ text, textContains }) => {
+  const base = '(//a | //*[@role="button"] | //button)'
+  if (text) {
+    return `${base}[normalize-space(.)="${text}" or @aria-label="${text}"]`
+  } else if (textContains) {
+    return `${base}[contains(normalize-space(.),"${textContains}") or contains(@aria-label,"${textContains}")]`
+  }
 }
 
-const exactClick = async (page, text) => {
-  return (await exactlyFindClickable(page, text)).click()
-}
-
-const findClickable = (page, text) => {
-  return page.waitForXPath(`(//a | //*[@role="button"] | //button)[contains(normalize-space(.),"${text}") or contains(@aria-label,"${text}")]`)
-}
-
-const click = async (page, text) => {
-  return (await findClickable(page, text)).click()
+const click = async (page, xpath) => {
+  return (await page.waitForXPath(xpath)).click()
 }
 
 const findText = (page, text) => {
```

Comment:
As `findText` is currently implemented,`text` should be `textContains` to be consistent with the current naming convention. Ultimately it should be consistent with `clickable` and `svgText` (and `labelContains` for `input` and `select`).

Change request note: This is the only inconsistency I noticed that we should definitely fix in this PR. I have not looked at more than the diff context that Github shows me so it's possible that there are others.